### PR TITLE
✨ Use correct aliases for capm3 api

### PR DESCRIPTION
**Pull request by fmuyassarov**
**What this PR does / why we need it**:
We don't have v1 APIs yet, so let's be explicit.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -23,8 +23,8 @@ import (
 	"time"
 
 	bmoapis "github.com/metal3-io/baremetal-operator/apis/metal3.io/v1alpha1"
-	infrav1alpha4 "github.com/metal3-io/cluster-api-provider-metal3/api/v1alpha4"
-	infrav1 "github.com/metal3-io/cluster-api-provider-metal3/api/v1alpha5"
+	infrav1alpha5alpha4 "github.com/metal3-io/cluster-api-provider-metal3/api/v1alpha4"
```
Comment (furkatgofurov7) [inline main.go:26]: typo

File: main.go [hunk 2]
```diff
@@ -58,8 +58,8 @@ var (
 func init() {
 	_ = scheme.AddToScheme(myscheme)
 	_ = ipamv1.AddToScheme(myscheme)
-	_ = infrav1.AddToScheme(myscheme)
-	_ = infrav1alpha4.AddToScheme(myscheme)
+	_ = infrav1alpha5.AddToScheme(myscheme)
+	_ = infrav1alpha5alpha4.AddToScheme(myscheme)
```
Comment (furkatgofurov7) [inline main.go:62]: same

**Review by furkatgofurov7 — CHANGES_REQUESTED** [review]

File: main.go (hunk 1 above)
Comment (fmuyassarov) [reply main.go]: thanks, fixed

**Review by furkatgofurov7 — COMMENTED** [review]
Since this is just a cosmetic change, approving. 

/approve

**Comment by furkatgofurov7** [comment]
/retitle ✨ Use correct aliases for capm3 api

**Review by lentzi90 — APPROVED** [review]

**Pull request merged**